ios/RNGoogleSignin.m

```diff
@@ -96,10 +96,12 @@ - (dispatch_queue_t)methodQueue
 {
   UIViewController* presentingViewController = RCTPresentedViewController();
   NSString* hint = options[@"loginHint"];
+  NSArray* additionalScopes= options[@"scopes"];
```

Comment:
hi, sorry I didn't make myself clear, but we must not change the library api: the scopes are already passed to the `configure` call and we should save them as an instance variable which we will use here. 

The parameters of the `signIn` call will remain the same. Hope this makes sense, thank you

Reply:
yes but `@Property (nonatomic) NSArray *_scopes;` would be `@Property (nonatomic) NSArray *scopes;`

can you please make the change, test it out and let me know it works? Thank you @nikhilsoni-stack